backend/src/main/java/com/project/stocks/dto/CategoryList.java

```diff
@@ -0,0 +1,22 @@
+package com.project.stocks.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class CategoryList {
+
+    public CategoryList() {
+    }
+
+    @JsonProperty("category")
+    private List<String> category;
```

Comment:
This class is a list of strings. Does not need to exist

Reply:
Done